src/wallet/wallet2.h

```diff
@@ -701,15 +767,17 @@ namespace tools
   };
 }
 BOOST_CLASS_VERSION(tools::wallet2, 18)
-BOOST_CLASS_VERSION(tools::wallet2::transfer_details, 7)
+BOOST_CLASS_VERSION(tools::wallet2::transfer_details, 9)
```

Comment:
Because I squashed two patches into one, both increasing the version. So nobody but me will ever see v8 :) Guess I can set it to 8 before merging.